pkg/services/datasources/models.go

```diff
@@ -147,6 +168,32 @@ type DeleteDataSourceCommand struct {
 	UpdateSecretFn UpdateSecretFn
 }
 
+// Correlations are uniquely identified by a SourceUid+TargetUid pair.
```

Comment:
I thought we were going to generate another id f or this, since it would be possible to have the same source and target?

Reply:
leftorver comment, will update. thanks for spotting this.